### DISCUSSION TITLE
Tropo event October 2023

### DISCUSSION THREAD

**Discussion by g7ruh** [jvde-github/AIS-catcher#242] [General]
During October 2023 I captured a long-lived tropo event.

It lasted from 7-10 October. The high pressure was steady and moved little during this time. This shows what it was like on 9th October:

![isobaric-chart-202310091200Z](https://github.com/jvde-github/AIS-catcher/assets/5949609/db545f92-f93c-44c6-b202-0e7081b27610)

My normal reception distance is 30-50 NM, sometimes enhanced conditions give 100-120 NM. This time it was much more.

This is from 7th Oct 0900 GMT, about 170 NM to south-west:

![AIS-catcher-big-lift-202310071000](https://github.com/jvde-github/AIS-catcher/assets/5949609/4bffb426-53e7-4e15-8b13-81e73886a14a)

and 3 hours later:

![image](https://github.com/jvde-github/AIS-catcher/assets/5949609/b0fa51ff-0fc2-4562-8157-0bfe56fab013)

Distance chart from 7th at 1926 GMT. You can see that distance started to increase about 20 hours before:

![AIS-catcher-big-lift-Distance-202310072026](https://github.com/jvde-github/AIS-catcher/assets/5949609/b1e80c7d-0434-4a7b-be91-7a74a8b4227d)

The following day (8th) at 0835 GMT:

![AIS-catcher-big-lift-202310080935](https://github.com/jvde-github/AIS-catcher/assets/5949609/55ac28c3-9c33-4732-b5f4-1a98414476b3)

It was much the same at 1854 GMT the same day, picture not included.

The following morning it was much the same at 0704 GMT

![AIS-catcher-big-lift-202310090804](https://github.com/jvde-github/AIS-catcher/assets/5949609/eb6f7bfd-3782-4faf-bd92-c8621837c4d6)

The rest of the day conditions dropped back a bit and were more like they were on 7th October, then on 10th October at 0658 GMT
things got better: 

![AIS-catcher-big-lift-202310100758](https://github.com/jvde-github/AIS-catcher/assets/5949609/26fc5ef7-0d35-4ce4-b504-f7763fb7c554)

and better, at 0849 GMT: note the message count for OOCL Felixstowe:

![AIS-catcher-big-lift-202310100949-OOCL Felixstowe 566 NM](https://github.com/jvde-github/AIS-catcher/assets/5949609/91f2da74-fbfe-48af-b767-29aeafed24b3)

This is the end of the event, messages per hour graph time was 10th October 1511 GMT and messages started to drop off about 15 hours before:

![image](https://github.com/jvde-github/AIS-catcher/assets/5949609/f0967658-ee44-47ad-b122-7e84e641089e)

So AIS-catcher can certainly be used to detect tropo events. 560 NM is certainly not bad. I was not watching the screens all the time, so there may have been a greater distance received. That is where some kind of automation could help.

My normal AIS-catcher web screens are two local ones, around Southampton Docks and Isle of Wight. Two others showing English Channel about 120 NM east and west, the final one is the same as the picture above with the OOCL Felixstowe detail panel.

A quick glance at the last two gives a fair idea of reception conditions. I often see as far as Calais and sometimes as far as The Netherlands, but not as good as last October!

Roger G7RUH

**Comment by jvde-github** [comment]
Super interesting. If we have a bit more critical mass with feeders, a tropo map based on AIS reception could be interesting.

**Reply by MaxX850** [reply]
Hi, I live on the Baltic Sea coast.  If I started sending data from my station to aiscatcher.org, you would see that this station has a range of 800 to 2000 NM.  However, this is not tropo but Topol - a Russian electronic warfare system that jams GPS from the Kaliningrad Oblast on a large area of ​​the Baltic Sea and in several countries in Central Europe and Scandinavia.  Sometimes a ship mooring 1km from my station appears off the coast of Scotland or Bosnia and Herzegovina and returns to my house after a few minutes.

**Reply by jvde-github** [reply]
Oh wow. I would be interested in having a look at some of this raw historical data at some point. I am interested in detecting spoofing and jamming algorithmically to see if we can infer the vessel position from other data.

**Reply by g7ruh** [reply]
"Sometimes a ship mooring 1km from my station appears off the coast of Scotland or Bosnia and Herzegovina"

@MaxX850 Interesting. I have seen several vessels report a strange position (e.g. Scotland or The Netherlands) and I recall seeing them in Southampton (my local port). When I look, that vessel moored in Southampton but I have reports from further away. I started to think this was "single bit errors" and may well be, but you add a different dimension to this if the vessel's GPS is being "modified" by jamming. I guess jamming is also subject to tropo enhancement.

Thank you for the reply and I am sorry you suffer from that sort of interference, most frustrating.

Regards
Roger G7RUH

**Reply by MaxX850** [reply]
![Screenshot_20240219_215043_Chrome](https://github.com/jvde-github/AIS-catcher/assets/46927569/8004ea7b-315d-42e3-abff-f7be6c8a46f1)
I have been interested in digital broadcasts for many years.  I have been receiving AIS for over 10 years.  During this time, I used various hardware and software.  It happened to me that I picked up the position of a ship that was really far away - off the coast of Norway or Denmark, but these positions could be confirmed on websites such as fleetmoon or MT based on photos uploaded from that place or port calls.  However, since last year I have started observing such short changes in the position of individual units, reaching several hundred nautical miles.  Sometimes even several thousand NM.  I also have a commercial AIS receiver - I need to turn it on and check if it will also record such jumps.  Some programs, such as shipplotter, have a filter that can limit the recording of unit positions above a user-defined distance from the station.  I assume that large websites such as MT and others can also filter such data uploaded by users.  I'm not sure if what I'm seeing is GPS signal jamming, but my region is listed in warnings for airline pilots as being at risk for this type of interference. Also on flightradar24 I can see strange things.

**Reply by MaxX850** [reply]
![Screenshot_20240219_215525_Chrome](https://github.com/jvde-github/AIS-catcher/assets/46927569/40fbad54-d439-4960-be95-55f2862c346e)
Here I can see a medium gps interference at my home location.

**Reply by Phil353556** [reply]
You're right, I also follow adsb data and the website (your screenshot) is very interesting: https://gpsjam.org/ I discovered it a few months ago.
Kind regards
Philippe

**Comment by g7ruh** [comment]
@MaxX850 

Thanks, your Marco Polo picture certainly looks like a position being reported as good data from a GPS but a vessel cannot move that fast or over land! I saw that sort of thing on a smaller scale when GPS was subject to Selective Availability "errors" years ago. I was researching / testing amateur radio APRS accuracy at the time.

Here is an example which triggered my curiosity. I put it down to the "single bit error" type of issue at the receiving web site, clearly wrong but interesting to me. Screen grabs taken at the same time. One web site wrong the other OK and agreed with my AIS.


![AIS-Regal-Princess-Screenshot 2023-07-07 133019](https://github.com/jvde-github/AIS-catcher/assets/5949609/edfff55f-adfa-4fbb-a36e-5b463a923bbe)

![AIS-Regal-Princess-Southampton Dock-Screenshot 2023-07-07 133019](https://github.com/jvde-github/AIS-catcher/assets/5949609/8794ef8d-1f08-4af9-aef2-806e8adc6ea4)

So not like yours, but a strange event.

Regards
Roger

**Reply by MaxX850** [reply]
Over 400 ships... impressvie! I also upload data to MT and Vesselfinder.  VS basically shows what the AIS-catcher receives, but MT uses a strange selection policy if there are several AIS stations operating in the area - my range is about 10NM and an average of 85-110 ships, but on MT the range is only 2-3NM and 20 ships.  As I mentioned earlier, I will try to run another AIS receiver and see if it also registers such strange behavior.

**Reply by jvde-github** [reply]
If you want to and are open to it, you are more than welcome to run with -X. The jamming is not an issue as we show raw AIS data including any distortions. In fact, if it is jamming it makes it more interesting.

I am building a function which shows for a vessel, who receives what and how. This could be interesting to investigate these issues.

**Reply by MaxX850** [reply]
Thanks!  In the next few days I plan to upgrade to version 0.57 on my raspberrypi and then I will add the -X option in the configuration file. I also need to check how to set the station name and what else is required to upload data to aiscatcher.org

**Reply by jvde-github** [reply]
Ah great! So far only -X is needed, you cannot set a station name yet (for those who want) but will be possible in the next few months. So far it’s anonymous.

**Reply by MaxX850** [reply]
Ok. I'll try uploading anyway :)

**Reply by reneselander** [reply]
About Marine Traffic ( MT ), I contacted them because I was frustrated about their policy regarding which stations to be prioritized. My station performs well and Vesselfinder, ShipXplorer and aiscatcher.org presents my data proper with actual number of ships ( usually around 40 - 60 class A ships ). MT shows usually less than 5 ships from my station which give my station bad statistic and status "Low Coverage".

Here is the answer I got from MT Support:

"The problem here is not with your station, but with the fact that the area is covered by a majority of stations.
The number of stations that comprise the MarineTraffic Network keeps expanding over time and, as expected, there are areas of the world that get covered by two or more stations at the same time. Under this perspective, whenever such coverage overlaps occur, this is how the MarineTraffic platform manages the situation: 

Data gets downsampled server-side to one position report per MMSI per minute to handle the increased load. The contact report with the most recent network timestamp is taken into account by the server. If there are still duplicate records, the station with the fastest report is accredited on a "first come first serve" basis.
 
This results in having discrepancies like the one that you are seeing.
 
We have plans on updating this logic and it is on the roadmap but it is not still clear when this will be implemented."

**Reply by jvde-github** [reply]
That is interesting! Makes the stats a bit useless though for managing your station. My hypothesis was that they prioritized their own equipment as I was experiencing something similar. Did not expect this logic…

**Reply by g7ruh** [reply]
That explains what I see here where there are quite a few feeders to MT and I sometimes check a vessel on MT to make sure my feed is working OK, sometimes it is hard to find a vessel with my feeder as the reporting station. First come first served explains a lot! 

Thanks you for posting the response from MT

**Reply by MaxX850** [reply]
@jvde-github
"Ah great! So far only -X is needed, you cannot set a station name yet (for those who want) but will be possible in the next few months. So far it’s anonymous"

Ok, update to 0.57 and feeding hub light on :)

**Reply by jvde-github** [reply]
I can see your feed coming in indeed on aiscatcher.org/hub. I am developing a way to see for a vessel what messages came in, who received it and when. That could be interesting if there are distortions again.

Thanks for sharing and the background!

**Reply by MaxX850** [reply]
Most false reports are generated by a single ship.  "Taurus" is a port tug.  I wonder if it's a problem with his GPS receiver or actual interference.  I also need to look at other units moored in the port of Gdańsk.
![Screenshot_20240221_164548_Chrome](https://github.com/jvde-github/AIS-catcher/assets/46927569/b5f9574a-0a67-47a3-a814-a7ce9692763a)

**Reply by jvde-github** [reply]
Is that not the location of the vessel with sign of latitude changed? As it is such a vertical line…

**Reply by jvde-github** [reply]
No probably it is zero…. And other coordinate the same

**Reply by jvde-github** [reply]
I suspect it is a bug in the transmitter software? Could still be jamming in other scenarios btw

**Reply by jvde-github** [reply]
![IMG_4494](https://github.com/jvde-github/AIS-catcher/assets/52420030/71808a39-d8e3-4013-982f-936acc733af1)
There is a ship in Seattle that always jumps around in the same way. Easy to exclude but struggle to understand why…

**Reply by MaxX850** [reply]
In the position of this unit, the latitude most often changes, but sometimes the longitude also changes.

**Reply by jvde-github** [reply]
Since only one coordinate frequently changes it is not likely to be jamming or gps signal distortion. You would expect more randomness

**Reply by MaxX850** [reply]
![Screenshot_20240220_095619_Chrome](https://github.com/jvde-github/AIS-catcher/assets/46927569/498992b5-09c8-4090-85d8-f340bb0d6cb0)
![Screenshot_20240219_220842_Chrome](https://github.com/jvde-github/AIS-catcher/assets/46927569/c40e7b17-7c74-4e51-9a1d-421f74291fb2)
That's some screenshots from 19.02 and 20.02. Unfortunately I didn't check what ship (or ships) was that.

**Reply by jvde-github** [reply]
If there is again such distortion can you dump the NMEA line here? You can right click and do last message. Please the text and not an image 🙃

**Reply by MaxX850** [reply]
I'm at work now.  I'll do it when I get home.

**Reply by jvde-github** [reply]
No rush can be any time

**Reply by MaxX850** [reply]
![Screenshot_20240224_204555_Chrome](https://github.com/jvde-github/AIS-catcher/assets/46927569/72f15223-1c3e-46ea-9da5-1c01fa0fd25d)
Until now, it seemed to me that most of these types of incidents were caused by one ship - Taurus, a small port tug.  However, now I caught something much bigger - Mogens Maersk, a 400-meter long container ship, jumping around the port basin and sometimes even reaching the coast of Great Britain.  It's hard for me to dump a NEMA message with an incorrect location because these jumps are very short and I usually only see them when I turn on the ship's route display.

**Reply by jvde-github** [reply]
Thanks! Maybe I can find something in the data stream tomorrow. Thanks for flagging

**Reply by jvde-github** [reply]
I see the same jumps also with other aggregators, so that makes it less likely that it is on the receiver end but something with the location broadcasted by this sender. Makes it difficult to pinpoint.

**Reply by jvde-github** [reply]
Another explanation I am pondering over whether there is a bug that mixes up the ordering of the path points. The points outside the port is an anchorage area where the ship might have been at anchor.

![image](https://github.com/jvde-github/AIS-catcher/assets/52420030/a9afab26-16f2-4be3-b838-34a58c2bc7bc)

**Comment by g7ruh** [comment]
Update April 13th 2024

Jasper has changed the number of vessels pop-up to separate Class A and B (thanks Jasper).

This does two things at this location: 

1) it makes it easier to see when there is a major sailing event taking place: the number of Class B (pleasure craft) under way is near or exceeds the number moored.

2) it makes it easier to see enhanced reception conditions as the number of Class A vessels under way is significantly higher than average.

![AIS-catcher-tropo-202404131326](https://github.com/jvde-github/AIS-catcher/assets/5949609/b69588bf-7d26-4736-bd7e-26c501bb2b92)

The above, from today, gives an example of both 1) and 2).

Maybe the most simple way to detect enhanced reception conditions is to count the number of fixed structures. These by their nature do not move (!!). When the number goes up from the "normal level" which for me is 4 or 5 then it can trigger an alarm. If the average number of fixed structures over a month or more ( I think 3 months) is recorded, then an alarm can be triggered when the current number reaches (for example) 10 - 15 in my case, then I know the wider map is worth looking at.

If the total number of vessels circle changes from the white background to another colour (say red) when the condition is met then viewing the web page shows the change in an obvious way.

The examples from last October above have an even higher total of fixed structures.

If the average number is stored and the trigger point is provided as a parameter then the number can be adjusted as needed, but the average will not change too much over time as enhanced propagation does not normally last for more than a few days.

I think it is better than trying to monitor the number of Class A vessels as the number is always changing.

It would be helpful if the alarm colour is configurable, e.g. to help those who may be colour-blind.

Roger
